features/checkout/paying_for_order/changing_payment_method_after_order_confirmation.feature

```diff
@@ -32,3 +32,13 @@ Feature: Changing the method after order confirmation
         When I go to the change payment method page
         And I try to pay with "Offline" payment method
         Then I should see the thank you page
+
+    @api
+    Scenario: Changing the payment method for placed order
```

Comment:
I guess that you can adjust the existing scenarios for UI to be more business and the use them also for API context. What is more, it seems that in this api scenario you are missing some important behaviour from ui as paying by the changed payment method.